src/libs/OptionsListUtils.js

```diff
@@ -216,18 +216,19 @@ function isPersonalDetailsReady(personalDetails) {
 function getParticipantsOptions(report, personalDetails) {
     const participants = lodashGet(report, 'participantAccountIDs', []);
     return _.map(getPersonalDetailsForAccountIDs(participants, personalDetails), (details) => ({
-        keyForList: details.login,
-        login: details.login,
+        keyForList: details.login || details.displayName || String(details.accountID),
+        login: details.login || details.displayName,
```

Comment:
What would go wrong if we keep the login field empty/undefined? (instead of falling back to displayName)

Reply:
I don't think this fix is only for IOU pages 🤔  I'll look around and see if see if this bug exists in other pages showing options

Reply:
@s77rt can you explain your hesitation for using `displayName` as a backup here?

Reply:
I think the only other possibly better idea is to use `ReportUtils.getDisplayNameForParticipant` here as a backup

Reply:
Did you find where we set `login` as the `text`? I'm not sure we found the root cause yet. As far as I can tell we already set the `text` (for OptionRow) to `displayName`.

Reply:
Ooh `this.props.option.isMoneyRequestReport` is not `true`, which is why `shouldUseFullTitle` isn't true, which is why we're not showing the full title which includes the display name

Reply:
the issue is only reproducible in staging, in main it seems to work fine. so it's fixed already?

Reply:
@s77rt are you talking about 1 specific part or this whole PR?

Reply:
nvm, it's reproducible on main too, had to toggle staging switch

Reply:
This PR is up to date with `main` - and I was easily able to reproduce my screenshot above in this PR without my changes

Reply:
@s77rt how do you feel about my latest change - using `isMoneyRequestReport: true,` since we only use this in money request reports right now

Reply:
Not the correct fix. I think I got it. The bug is on OptionRow, `displayNamesWithTooltips` relies on `option.login`, and if it does not exist the displayNamesWithTooltips will be empty.

Reply:
testing

Reply:
Relaying on accountID instead of login seems to do it. 

```js
        const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(
            (this.props.option.participantsList || this.props.option.accountID ? [this.props.option] : []).slice(0, 10),
            isMultipleParticipant,
        );
```

Reply:
Ooh that seems like a better fix - though I'm still surprised `isMoneyRequestReport: true,` on options for MoneyRequest pages... Maybe that's a follow-up question

Reply:
I will be back in 45min or so, the above fix should be the correct fix, feel free to merge if this is critical

Reply:
isMoneyRequestReport is unrelated. this is true when the report is iou report or expense report, this has nothing to do with iou, it's used for other optionrow usage (not totally sure, but that's what I think)

Reply:
asking in slack for someone to test on all platforms for you